src/sources/http_client/client.rs

```diff
@@ -202,20 +201,15 @@ impl HttpClientContext {
 
     /// Enriches events with source_type, timestamp
     fn enrich_events(&self, events: &mut Vec<Event>) {
+        let now = Utc::now();
+
         for event in events {
             match event {
                 Event::Log(ref mut log) => {
-                    self.log_namespace.insert_vector_metadata(
-                        log,
-                        log_schema().source_type_key(),
-                        "source_type",
-                        Bytes::from(HttpClientConfig::NAME),
-                    );
-                    self.log_namespace.insert_vector_metadata(
+                    self.log_namespace.insert_standard_vector_source_metadata(
                         log,
-                        log_schema().timestamp_key(),
-                        "ingest_timestamp",
-                        Utc::now(),
+                        HttpClientConfig::NAME,
+                        now,
```

Comment:
This is a slight breaking change, but probably a good one - before each now could be very slightly different and now they wont. Is this intentional?

Reply:
Technically breaking, I suppose - updated for both all events having the same ingest time as well as slightly more performant (not having to syscal for each individual event).

Reply:
Technically breaking in a sense, but definitely more accurate: we ingested these events in a single batch, so they should all (reasonably, I'd say) bear the same timestamp for ingestion.

Beyond that, I agree with Spencer that avoiding looking up the time so often is also very beneficial.

Reply:
I don't know if I'd call this out as a breaking change in the release notes, but it could be called out as a bug fix? cc @jszwedko

Reply:
Agreed that this is a minor breakage that can just be listed as a bug fix.